mobile-widgets/qml/qmldir

```diff
@@ -0,0 +1,3 @@
+MapWidget ../../map-widget/qml/MapWidget.qml
+MapWidgetContextMenu ../../map-widget/qml/MapWidgetContextMenu.qml
+MapWidgetError ../../map-widget/qml/MapWidgetError.qml
```

Comment:
just for my understanding - this used to work because we bundled things as resources... why is this needed now?
My guess is so that if you point to the "in tree" directory things automagically work?

Reply:
A good question, in order to let others know as well I have added a comment in the file.

In short QQmlApplicationEngine allows imports to be from different directories, but Components (other qml files that you reference like e.g. SsrfButton needs to be in the same directory). The qrc file actually make the same aliases.

Reply:
excellent. thanks.